routes/error-tracker.js

```diff
@@ -102,7 +103,11 @@ function handler(req, res) {
   } else {
     errorType = errorType + '-1p';
   }
-  if (canary) {
+
+  // Do not append binary type if 'prod' since that is the default
+  if (binaryType && binaryType !== 'prod') {
```

Comment:
Technically correct, but I'd rather the `binaryType !== 'prod'` be in a nested if statement.

Reply:
done. PTAL